### ISSUE TITLE
ColorSpacesTest#testCorbisRGBSpecialHandling() fails with JDK8/LCMS

### ISSUE THREAD

**Issue by ankon** [haraldk/TwelveMonkeys#41]
When building with JDK 8 (b129) the ColorSpacesTest#testCorbisRGBSpecialHandling() fails:
~~~~
-------------------------------------------------------------------------------
Test set: com.twelvemonkeys.imageio.color.ColorSpacesTest
-------------------------------------------------------------------------------
Tests run: 18, Failures: 1, Errors: 0, Skipped: 0, Time elapsed: 0.067 sec <<< FAILURE!
testCorbisRGBSpecialHandling(com.twelvemonkeys.imageio.color.ColorSpacesTest)  Time elapsed: 0.007 sec  <<< FAILURE!
java.lang.AssertionError: array lengths differed, expected.length=452 actual.length=420
	at org.junit.Assert.fail(Assert.java:91)
	at org.junit.internal.ComparisonCriteria.assertArraysAreSameLength(ComparisonCriteria.java:72)
	at org.junit.internal.ComparisonCriteria.arrayEquals(ComparisonCriteria.java:36)
	at org.junit.Assert.internalArrayEquals(Assert.java:414)
	at org.junit.Assert.assertArrayEquals(Assert.java:200)
	at org.junit.Assert.assertArrayEquals(Assert.java:213)
	at com.twelvemonkeys.imageio.color.ColorSpacesTest.testCorbisRGBSpecialHandling(Unknown Source)
~~~~

This seems to be caused by the use of LCMS in JDK8, while earlier JDKs use kcms. The test can be made to pass by using the JVM arguments to switch back to kcms:
~~~~
-Dsun.java2d.cmm=sun.java2d.cmm.kcms.KcmsServiceProvider
~~~~

Unofrtunately I cannot get my JDK 1.7.0u45 to use LCMS to check in the other direction.

**Comment by haraldk** [comment]
Fixed in f588d65565cdc85299cd7446053329f460f206f9.

-- 
Harald K

**Comment by haraldk** [comment]
And better yet, in 94ed531fb284d824e6edbd40b3eb406366fe790f. :-)

-- 
Harald K
